models/order.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+from bson import ObjectId
+
+class OrderBase(BaseModel):
+    basketId: str
```

Comment:
We will need to add userID to orders as we might need to fetch all past and current orders of a single user
Also instead of adding the basketId we can add the products from the basket to the orders, as the basket and user will be a one-to-one relationship